src/pages/createProfile/createProfileSteps/Complete.tsx

```diff
@@ -0,0 +1,47 @@
+import { FunctionComponent } from "react";
+import { Box } from "@mui/material";
+import NEWMLogo from "assets/images/NEWMLogo";
+import { FilledButton, GradientTypography, Link, Typography } from "elements";
+
+const Complete: FunctionComponent = () => {
+  return (
+    <Box display="flex" flexDirection="column" alignItems="center">
+      <Box mb={ 4 } alignSelf="center">
+        <NEWMLogo />
+      </Box>
+
+      <Typography
+        align="center"
+        fontWeight="extra-bold"
+        variant="xxxl"
+        fontFamily="Raleway"
+      >
+        Aaaaand we&apos;re done.
+      </Typography>
+
+      <Box mt={ 1 } mb={ 6 }>
+        <GradientTypography
+          variant="xxxl"
+          fontFamily="DM Serif Text"
+          fontWeight="regular"
+          fontStyle="italic"
+        >
+          Shall we?
+        </GradientTypography>
+      </Box>
+
+      <FilledButton type="submit">Enter NEWM</FilledButton>
```

Comment:
Should the be a submit button? I thought that at this point all info had been submitted?

Reply:
We could technically submit information to the API at each step, but then we should probably add redirects so that someone is brought to the correct step in the form if they quit out before completing it. Since it's only 3 steps, it seemed easier to just submit everything at the end.